app/src/organizers/models.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from users.models import User
+from conferences.models import Zosia
+
+def validate_phone_number(value):
+    phone_reg = r'\d{9}'
```

Comment:
I'd also allow formats with separators (i.e. `123-456-789`, `123 456 789`) and country code at the beginning (i.e. `+48`).